apps/wallet-mobile/src/features/Send/useCases/ConfirmTx/FailedTx/FailedTxScreen.tsx

```diff
@@ -5,20 +5,13 @@ import {Button, Spacer, Text} from '../../../../../components'
 import {useBlockGoBack} from '../../../../../navigation'
 import {COLORS} from '../../../../../theme'
 import {useNavigateTo} from '../../../common/navigation'
-import {useSend} from '../../../common/SendContext'
 import {useStrings} from '../../../common/strings'
 import {FailedTxImage} from './FailedTxImage'
 
 export const FailedTxScreen = () => {
   useBlockGoBack()
   const strings = useStrings()
   const navigateTo = useNavigateTo()
-  const {resetForm} = useSend()
-
-  const goToStart = () => {
-    resetForm()
```

Comment:
@jemubm I think only by removing the `resetForm` and moving the user to `startTx` it won't crash anymore, and the user can continue with the current data of the transaction, can you test it?  just by removing this line?